docs/manual/file_triggers.md

```diff
@@ -32,7 +32,7 @@ directories, symlinks etc.
 
 The file triggers are defined in spec files of packages. E.g. file trigger executing `ldconfig` could be defined in glibc package.
 
-Similarly to regular triggers, file trigger scripts (except the `%transfiletrigger*` variants) are passed the number of instances of the triggered and triggering package (as the `$1` and `$2` arguments, respectively) that will remain when the file trigger has completed.  This can be useful in order to perform a certain operation when either of the packages is updated or removed.
+Similarly to regular triggers, file trigger scripts are passed the number of instances of the triggered and triggering package (as the `$1` and `$2` arguments, respectively) that will remain when the file trigger has completed, as well as which of those two packages has activated the trigger (the `$3` argument).  As an exception, the `%transfiletrigger*` variants are always passed the value of -1 as the `$2` argument.
```

Comment:
This should state the actual values passed to $3.

Reply:
Yep... I just didn't feel like copy&pasting the same text from the trigger page but $1 and $2 are described here, and so should $3, so ack, will fix.

FWIW, this will be refactored anyway as part of the (planned) consolidation of the two trigger flavours into one page.